federatorai/assets/dashboard/overview.json

```diff
@@ -0,0 +1,229 @@
+{
+    "notify_list": [],
+    "description": "You can visualize Kafka Acceleration in ProphetStor Federator.ai Kafka Integration dashboard.[Federator.ai/version:v4.2.784]",
+    "author_name": "ProphetStor Data Services, Inc.",
+    "template_variables": [
+        {
+            "default": "*",
+            "prefix": "kube_cluster",
+            "name": "kube_cluster"
+        },
+        {
+            "default": "*",
+            "prefix": "kube_namespace",
+            "name": "kube_namespace"
+        },
+        {
+            "default": "*",
+            "prefix": "kube_deployment",
+            "name": "kube_deployment"
+        },
+        {
+            "default": "*",
+            "prefix": "topic",
+            "name": "topic"
+        },
+        {
+            "default": "*",
+            "prefix": "consumer_group",
+            "name": "consumer_group"
+        }
+    ],
+    "is_read_only": false,
+    "id": "i5f-j7k-9u5",
+    "title": "ProphetStor Federator.ai Kafka Integration",
+    "url": "/dashboard/i5f-j7k-9u5/prophetstor-federatorai-kafka-integration",
+    "created_at": "2020-06-02T14:59:31.676822+00:00",
+    "modified_at": "2020-06-04T09:01:38.540851+00:00",
+    "author_handle": "support@prophetstor.com",
+    "widgets": [
+        {
+            "definition": {
+                "yaxis": {
+                    "max": "auto",
+                    "label": "",
+                    "scale": "linear",
+                    "min": "auto",
+                    "include_zero": true
+                },
+                "title": "Federator.ai Recommended Replicas vs Current Replicas and Desired Replicas over Deployment",
```

Comment:
This title is a bit long and doesn't fit in the current width of the widget, can we make it shorter?